TodoListService/Controllers/TodoListController.cs

```diff
@@ -155,15 +159,15 @@ public IEnumerable<TodoItem> Get()
                 }
                 catch (AdalException ex)
                 {
-                    if (ex.ErrorCode == "temporarily_unavailable")
+                    if (ex.ErrorCode == SERVICE_UNAVAILABLE)
```

Comment:
good catch. This needs to be done in all the .NET samples, BTW. They all do that